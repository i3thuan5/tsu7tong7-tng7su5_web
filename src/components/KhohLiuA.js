import React from 'react'
import { ButtonStack, CopyButton } from 'demo-ui'


const KhohLiuA = ({漢字, 台羅, POJ}) => (
  <ButtonStack>
   { 漢字 ? <CopyButton 複製內容={漢字} 按鈕名="漢字"/> : null }
   { 台羅 ? <CopyButton 複製內容={台羅} 按鈕名="台羅"/> : null }
   { POJ ? <CopyButton 複製內容={POJ} 按鈕名="POJ"/> : null }
  </ButtonStack>
)

export default KhohLiuA
