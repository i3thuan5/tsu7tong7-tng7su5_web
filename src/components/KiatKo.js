import React from 'react'
import { Block } from 'demo-ui'
import HianSiKhohLiuA from '../containers/HianSiKhohLiuA'
import SinSit from './SinSit'
import Bun from './Bun'


const KiatKo = ({查詢結果, 錯誤訊息}) => {
  return (
  <Block>
    {
      查詢結果 ? <HianSiKhohLiuA/>
      : null
    }
    <SinSit 訊息={錯誤訊息}/>
    <Bun 查詢結果={查詢結果}/>
  </Block>
  )
}

export default KiatKo;
