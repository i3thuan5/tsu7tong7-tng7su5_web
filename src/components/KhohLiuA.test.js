import React from 'react';
import renderer from 'react-test-renderer'
import { CopyButton } from 'demo-ui'
import KiatKo from './KiatKo';
import KhohLiuA from './KhohLiuA'


it('空結果無複製鈕仔', () => {
  const tree = renderer
    .create(<KhohLiuA 漢字='' 台羅='' POJ=''/>)
    .toJSON()
  expect(tree).toMatchSnapshot();
});
