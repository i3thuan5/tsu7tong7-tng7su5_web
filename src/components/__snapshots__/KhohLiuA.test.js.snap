// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`空結果無複製鈕仔 1`] = `
<div
  className="app ui stackable one large teal buttons"
/>
`;
